.github/workflows/test.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-12, macos-11, ubuntu-20.04, windows-2019, windows-2022]
-        go: ["1.16", "1.17", "1.18", "1.19"]
+        os: [macos-12, macos-13, ubuntu-22.04, windows-2019, windows-2022]
+        go: ["1.16", "1.17", "1.18", "1.19", "1.20", "1.21"]
```

Comment:
Maybe we could remove some Go releases here, even though it works on older Go versions (make CI faster).

Reply:
yep, makes sense